Integrations/integration-Aella_StarLight.yml

```diff
@@ -0,0 +1,331 @@
+            event = '*'
+
+        score = demisto.getParam('severity')
+        if score:
+            try:
+                score = int(score)
+                if score < 0 or score > 100:
+                    score = 50
+            except:
+                score = 50
+        else:
+            # Default score
+            score = 50
+
+        index_str = 'aella-ser*'
+
+        query_str = 'event_name:{}  AND severity:>{}'.format(event, score)
+        ts_str = str(long(lastRunTime * 1000))
+        query_json = { 'query':
+                            { 'bool':
+                                { 'must':
+                                    [ { 'query_string':
+                                        {
+                                          'query': query_str,
+                                          'analyze_wildcard': True
+                                        }
+                                      },
+                                      { 'range':
+                                         { 'timestamp':
+                                             { 'gt': ts_str }
+                                         }
+                                      }
+                                    ]
+                                }
+                            }
+                     }
+        end_point = URL + '/{0}/{1}/_search'.format(index_str, 'amsg')
+
+        action_result = {}
+        make_rest_call(end_point,
+            USERNAME, PASSWORD, action_result, data=query_json
+        )
+
+        if action_result['status'] == 'Success':
+            demisto.info('Poll incidents ok')
+            data = action_result.get('data')
+            #demisto.info('Data is {}'.format(data))
+            if not isinstance(data, dict):
+                demisto.error('Data returned in wrong format {}'.format(data))
+                demisto.incidents([])
+                return
+            hits = data.get('hits', {}).get('hits', [])
+            incidents = []
+            for hit in hits:
+                source = hit.get('_source', None)
+                if not source:
+                    continue
+                event_name = source.get('event_name', None)
+                try:
+                    event_severity = int(source.get('severity', None))
+                    if event_severity > 75:
+                        severity = 'high'
+                    elif event_severity > 50:
+                        severity = 'medium'
+                    else:
+                        severity = 'low'
+                except:
+                    severity = 'unknown'
+
+                if not event_name:
+                    continue
+                eid = hit['_id']
+                sdi = '{}_{}'.format(event_name, eid)
+                incident = {
+                    'name' : sdi,
+                    'label' : 'Starlight event',
+                    'aella_eid' : eid,
+                    'aella_event' : event_name,
```

Comment:
This field is unnecessary here, it is enough to pass it in the `rawJSON`.

Reply:
same as above

Reply:
done